### PR TITLE
[zh-cn]: Add Chinese document to the HTMLImageElement: complete property

### PR DESCRIPTION
**Pull request by Saraph1nes**
[zh-cn]: 为HTMLImageElement: complete 属性增加中文文档

### CODE REVIEW

File: files/zh-cn/web/api/htmlimageelement/complete/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: "HTMLImageElement: complete 属性"
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmlimageelement/complete/index.md:2]: 使用全角冒号，去掉引号

```suggestion
title: HTMLImageElement：complete 属性
```

File: files/zh-cn/web/api/htmlimageelement/complete/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+short-title: complete
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmlimageelement/complete/index.md:3]: 短标题无变化可忽略

```suggestion
```

File: files/zh-cn/web/api/htmlimageelement/complete/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+slug: Web/API/HTMLImageElement/complete
+l10n:
+  sourceCommit: a3d9f61a8990ba7b53bda9748d1f26a9e9810b18
+---
+
+{{APIRef("HTML DOM")}}
+
+只读 {{domxref("HTMLImageElement")}} 接口的 **`complete`** 属性是一个布尔值，用于指示图像是否已完全加载。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmlimageelement/complete/index.md:11]:
```suggestion
{{domxref("HTMLImageElement")}} 接口的 **`complete`** 只读属性是一个布尔值，用于指示图像是否已完全加载。
```

File: files/zh-cn/web/api/htmlimageelement/complete/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+如果满足以下任何一个条件，则认为图像已经加载完毕：
+
+- 没有指定 [`src`](/zh-CN/docs/Web/HTML/Element/img#src) 和 [`srcset`](/zh-CN/docs/Web/HTML/Element/img#srcset) 属性。
+- `srcset` 属性不存在且 `src` 属性被指定为空字符串（`""`）时。
+- 图片资源已经完全获取，并且已经排队等待渲染/合成。
+- 图像元素已预先确定，图像已完全可用并准备好使用。
+- 图像损坏，即由于错误或图像加载被禁用导致图像加载失败。
+
+值得注意的是，由于图像可能是异步接收的，因此`complete`的值可能在脚本运行时发生变化。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/htmlimageelement/complete/index.md:25]:
```suggestion
值得注意的是，由于图像可能是异步接收的，因此 `complete` 的值可能在脚本运行时发生变化。
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

File: files/zh-cn/web/api/htmlimageelement/complete/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+一个布尔值，如果图像加载完成，则为 `true`；否则，值为 `false`。
+
+如果满足以下任何一个条件，则认为图像已经加载完毕：
+
+- 没有指定 [`src`](/zh-CN/docs/Web/HTML/Element/img#src) 和 [`srcset`](/zh-CN/docs/Web/HTML/Element/img#srcset) 属性。
+- `srcset` 属性不存在且 `src` 属性被指定为空字符串（`""`）时。
+- 图片资源已经完全获取，并且已经排队等待渲染/合成。
+- 图像元素已预先确定，图像已完全可用并准备好使用。
+- 图像损坏，即由于错误或图像加载被禁用导致图像加载失败。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/htmlimageelement/complete/index.md:22]:
```suggestion
- 图像“损坏”，即由于错误或图像加载被禁用导致图像加载失败。
```

File: files/zh-cn/web/api/htmlimageelement/complete/index.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+值得注意的是，由于图像可能是异步接收的，因此 `complete` 的值可能在脚本运行时发生变化。
+
+## 示例
+
+请思考一个图库应用程序，它具有将图片打开为灯箱模式的功能，以便更好地查看和编辑图像。这些照片可能非常大，所以你不想等待它们加载，所以你在代码中使用 `async`/`await` 在后台加载图像。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/htmlimageelement/complete/index.md:28]:
```suggestion
假设有这么一个图库应用程序，它能够以灯箱模式打开图像，以便更好地查看和编辑图像。这些照片可能非常大，所以你不想等待它们加载，因此你在代码中使用 `async`/`await` 在后台加载图像。
```

File: files/zh-cn/web/api/htmlimageelement/complete/index.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+但是想象一下，你有其他代码仅仅需要在图像完成加载后运行，例如对灯箱中的图像执行红眼消除的命令。理想情况下，如果图像尚未完全加载，则不会执行此命令，但为了提高可靠性，你需要检查以确保是这种情况。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/htmlimageelement/complete/index.md:30]:
```suggestion
但是想象一下，你还有其他一些仅需要在图像完成加载后运行的代码，例如对灯箱中的图像执行红眼消除的命令。理想情况下，如果图像尚未完全加载，则不会执行此命令，但为了提高可靠性，你需要检查以确保情况确实如此。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions:

**Comment by Saraph1nes** [comment]
> Some suggestions:

thanks for your suggestion 🙂

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Good job! @Saraph1nes 👍🏼

**Pull request merged**